pkgs/by-name/bi/bicep-lsp/package.nix

```
@@ -0,0 +1,60 @@
+{
+  autoPatchelfHook,
+  buildDotnetModule,
+  coreutils,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  icu,
+  lib,
+  libkrb5,
+  openssl,
+  stdenv,
+}:
+
+buildDotnetModule rec {
+  pname = "bicep-lsp";
+  version = "0.34.44";
+
+  src = fetchFromGitHub {
+    owner = "Azure";
+    repo = "bicep";
+    rev = "v${version}";
+    hash = "sha256-vyPRLPTvQkwN7unlIHs6DvpjXnXyW1PDtH9hhIOgN1A=";
+  };
+
+  projectFile = "src/Bicep.LangServer/Bicep.LangServer.csproj";
+
+  postPatch = ''
+    substituteInPlace global.json --replace-warn "8.0.406" "${dotnetCorePackages.sdk_8_0.version}"
+  '';
+
+  nugetDeps = ./deps.json;
+
+  # From: https://github.com/Azure/bicep/blob/v0.34.44/global.json#L7
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+
+  nativeBuildInputs = lib.optionals stdenv.isLinux [ autoPatchelfHook ];
+
+  buildInputs = lib.optionals stdenv.isLinux [
+    icu
+    libkrb5
+    openssl
+    stdenv.cc.cc.lib
+  ];
+
+  doCheck = !(stdenv.isDarwin && stdenv.isAarch64); # mono is not available on aarch64-darwin
+
+  meta = {
+    description = "Domain Specific Language (DSL) for deploying Azure resources declaratively";
+    homepage = "https://github.com/Azure/bicep/";
+    changelog = "https://github.com/Azure/bicep/releases/tag/v${version}";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ ] ++ lib.teams.stridtech.members;
+    platforms = [
+      "x86_64-linux"
+      "x86_64-darwin"
+      "aarch64-darwin"
+    ];
```

```suggestion
    platforms = lib.platforms.all;
    badPlatforms = [ "aarch64-linux" ];
```
Looks like aarch64-linux is a build target upstream, so it is a supported platform  https://github.com/Azure/bicep/blob/5273daf6f962dcd5299ee2623ae4edea802fe3c8/.github/workflows/build.yml#L34-L40

Would rather just mark it as a badPlatform on our end, atm.